go.mod

```diff
@@ -40,6 +41,8 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/mitchellh/reflectwalk v1.0.2
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/opentofu/opentofu v1.8.7
```

Comment:
Hmm this is a little suspect, likely we're not shadow-rewriting all imports in pkg/vendored/opentofu.

Reply:
Removed this. Fully using vendored (renamed) packages. No direct dependency on opentofu.